### DISCUSSION TITLE
hmm should we use "resource" in test_memusage instead of counting gc.get_objects() (answer: no)

### DISCUSSION THREAD

**Discussion by zzzeek** [sqlalchemy/sqlalchemy#8767] [Ideas]
this seems to be a very nice module and will spot all manner of memory leaks including in our external dependendies, cython code, etc.

**Comment by CaselIT** [comment]
Note that it's unix only: https://docs.python.org/3.10/library/unix.html

**Reply by zzzeek** [reply]
ah!  then forget it :)

**Reply by CaselIT** [reply]
Not sure if there is something comparable for windows. An alternative is to just have these tests on unix systems
